### DISCUSSION TITLE
无法使用PotPlayer播放视频

### DISCUSSION THREAD

**Discussion by InvoLve1** [AlistGo/alist#640] [Q&A]
![image](https://user-images.githubusercontent.com/93432245/156047348-c3db708f-34d2-40d9-bec5-73fcf3370595.png)

视频播放链接：https://alist.v2rayn.icu/d/%E6%9C%8D%E5%8A%A1%E5%99%A8%E6%96%87%E4%BB%B6%E5%A4%B9/%E7%BD%91%E7%AB%99%E8%A7%86%E9%A2%91/NARAKA%20%20BLADEPOINT%202022.02.27%20-%2001.07.08.06.DVR.mp4

**Comment by xhofe** [comment]
Error 521

**Reply by InvoLve1** [reply]
开启反向代理后就无法使用此视频播放器播放

**Reply by xhofe** [reply]
哪个链接。上面的链接文件不存在了
